src/jetstream/datastore/20170818162837_SetupSchema.go

```diff
@@ -10,6 +10,7 @@ import (
 func init() {
 	RegisterMigration(20170818162837, "SetupSchema", func(txn *sql.Tx, conf *goose.DBConf) error {
 		consoleConfigTable := "CREATE TABLE IF NOT EXISTS console_config ("
+		consoleConfigTable += "  auth_endpoint_type        VARCHAR(255)              NOT NULL, "
```

Comment:
We can't change a migration that has already been published - you'll need to move this to an ALTER statement in your new migration file.

Reply:
Thanks. Done. Will push with remaining changes.